README.md

```diff
@@ -34,23 +53,122 @@ They are said to be "bound" to the unit they are currently operating on.
+even if it had more instructions left to execute, same as in mvr. 
+
+#### cop
+
+Places the complement of the current base into the complementary strand, and enters "copy mode." Each base has its corresponding
+complement, known as the complementary base pairin (see [Definitions](#Definitions)). Copy mode remains in effect until an off 
+instruction is reached (or the end of the strand/enzyme). While in copy mode, any base that is "touched" by the enzyme will 
+have its complementary base placed above it in the complementary strand. When bases are copied into the complementary 
+strand, they are placed *upside-down* above the element copied, so that the strands are mirrored to each other. Copy mode applies 
+after a move, so that the movement off of a strand or into a gap between bases still preserves the behavior of the strand ending normally.  
+While in copy mode, additional cop instructions do nothing. 
+
+#### off
+
+Turns off copy mode. If copy mode is not enabled, then this does nothing. 
+
+#### ina
+
+Inserts an 'A' base into the position to the right of the enzyme, and if copy mode is on, then the base's complement (T) is placed above into
+the complementary strand. 
+
+#### inc
+
+Inserts an 'C' base into the position to the right of the enzyme, and if copy mode is on, then the base's complement (G) is placed above into
+the complementary strand. 
+
+#### int
+
+Inserts an 'T' base into the position to the right of the enzyme, and if copy mode is on, then the base's complement (A) is placed above into
+the complementary strand. 
+
+#### ing
+
+Inserts an 'G' base into the position to the right of the enzyme, and if copy mode is on, then the base's complement (C) is placed above into
+the complementary strand. 
+
+#### rpy
+
+Moves to the nearest pyrine to the right of this unit. (See [Definitions](#Definitions)). With copy mode enable, each base touched while sliding to the next pyrine 
+will be copied into the complementary strand. If there are no pyrines to the right of the current base, this does nothing.
+
+#### rpu
+
+Moves to the nearest purine to the right of this unit. (See [Definitions](#Definitions)). With copy mode enable, each base touched while sliding to the next purine 
+will be copied into the complementary strand. If there are no purines to the right of the current base, this does nothing.
+
+#### lpy
+
+Moves to the nearest pyrine to the left of this unit. (See [Definitions](#Definitions)). With copy mode enable, each base touched while sliding to the next pyrine 
+will be copied into the complementary strand. If there are no pyrines to the left of the current base, this does nothing.
+
+#### lpu
+
+Moves to the nearest purine to the left of this unit. (See [Definitions](#Definitions)). With copy mode enable, each base touched while sliding to the next purine 
+will be copied into the complementary strand. If there are no purines to the left of the current base, this does nothing.
+
+
+## Binding Preference
```

Comment:
What happens if a strand does not contain the base of an enzymes binding preference? It just doesn't act? Does that mean you can create strands that just won't operate on themselves and never will no matter how many iterations you run them for?

Reply:
Yes that is exactly correct, and how it is implemented. If an enzyme does not have an available base to bind to, then it will not act on the strand. It prints with an exit message too. There are a large number of 'dud' strands actually, that do nothing or simply loop behavior. 
i.e. AAAA, AGGG